analysis/topEFT/topeft.py

```diff
@@ -514,6 +531,29 @@ def process(self, events):
                     "appl_lst"     : ["isSR_2lSS" , "isAR_2lSS"] + (["isAR_2lSS_OS"] if isData else []),
                 },
             },
+  
+            "2l_4t" : {
+                "exactly_4j" : {
+                    "lep_chan_lst" : ["2lss_4t_p", "2lss_4t_m"],
+                    "lep_flav_lst" : ["ee" , "em" , "mm"],
+                    "appl_lst"     : ["isSR_2lSS" , "isAR_2lSS"] + (["isAR_2lSS_OS"] if isData else []),
+                },
+                "exactly_5j" : {
+                    "lep_chan_lst" : ["2lss_4t_p", "2lss_4t_m"],
+                    "lep_flav_lst" : ["ee" , "em" , "mm"],
+                    "appl_lst"     : ["isSR_2lSS" , "isAR_2lSS"] + (["isAR_2lSS_OS"] if isData else []),
+                },
+                "exactly_6j" : {
+                    "lep_chan_lst" : ["2lss_4t_p", "2lss_4t_m"],
+                    "lep_flav_lst" : ["ee" , "em" , "mm"],
+                    "appl_lst"     : ["isSR_2lSS" , "isAR_2lSS"] + (["isAR_2lSS_OS"] if isData else []),
+                },
+                "atleast_7j" : {
+                    "lep_chan_lst" : ["2lss_4t_p", "2lss_4t_m"],
+                    "lep_flav_lst" : ["ee" , "em" , "mm"],
+                    "appl_lst"     : ["isSR_2lSS" , "isAR_2lSS"] + (["isAR_2lSS_OS"] if isData else []),
+                },
+            },
```

Comment:
I don't think a new `2l_4t` dictionary would actually be necessary here. I think we could just include the `"2lss_4t_p"` and `"2lss_4t_m"` in the `lep_chan_lst` of the `2l` dictionary, right?

Reply:
Sure I can adapt that. What is the difference if you don't mind asking?

Reply:
I think the main difference is that it makes use of an already existing object (the 2l dictionary) instead of adding another dictionary that is essentially an exact copy of the already existing dictionary (just with different items in the `lep_chan_lst`). So from my perspective at least, it's more compact, and repeats less code, and is easier to maintain (since if we want to modify something about the dictionary in the future, we'd only have to do it in one place instead of two).

Reply:
Ok that's what I thought. Got it thanks!